### PR TITLE
Updated Readme with synology instructions

### PR DESCRIPTION
**Pull request by Khogniak**
:)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,3 +168,44 @@ The container exposes /config as a volume. This is the directory where the suppl
 If you have transmission authentication enabled and want scripts in another container to access and
 control the transmission-daemon, this can be a handy way to access the credentials.
 For example, another container may pause or restrict transmission speeds while the server is streaming video.
+
+## Make it work an a Synology NAS
```
Comment (haugene) [inline README.md]: "an a" -> "on"?

File: README.md [hunk 2]
```diff
@@ -168,3 +168,44 @@ The container exposes /config as a volume. This is the directory where the suppl
...
+- Check permission with `chmod 0755 TUN.sh`
+- Run it with `./TUN.sh`
+- Create the DNS config file by typing `vim /volume1/foldername/resolv.conf`
+- Paste the following lines :
+```
+nameserver 8.8.8.8
+nameserver 8.8.4.4
+```
+- Save the file with [escape] + `:wq!`
+- Create your docker container with `docker run --privileged -d -v /volume1/foldername/resolv.conf:/etc/resolv.conf -v /volume1/yourpath/:/data -e "OPENVPN_PROVIDER=PIA" -e "OPENVPN_CONFIG=Netherlands" -e "OPENVPN_USERNAME=XXXXX" -e "OPENVPN_PASSWORD=XXXXX" -p 9092:9091 haugene/transmission-openvpn -name TransmissionVPN`
```
Comment (haugene) [inline README.md]: Maybe say create container with "something like this" or similar. People are very prone to copy-pasting everything :)

**Comment by haugene** [comment]
Great stuff, thank you! I've commented on a couple of minor things. Feel free to update them if you'd like. I'll wait to see if you do, but then I'll merge it anyways :+1:

File: README.md [hunk 3]
```diff
@@ -168,3 +168,44 @@ The container exposes /config as a volume. This is the directory where the suppl
...
+- If you wish to access WebUI, just run the nginx container as explained in doc on ports 9091:9091
```
Comment (haugene) [inline README.md]: With the run command above, and nginx -p 9091:9091 as you say, they would have to change nginx.conf to:
* listen 9091;
* proxy_pass http://host.ip.address.here:9092;

Could be mentioned...

File: README.md (hunk 1 above)
Comment (Khogniak) [inline README.md]: Yes, sorry my english speaking/writing is improving everyday haha

File: README.md (hunk 3 above)
Comment (Khogniak) [inline README.md]: Ok, let just not mention the WebUI, it will be confusing with your explanation you made above in the Readme

**Comment by haugene** [comment]
Great! :+1:

**Pull request merged**